src/communication/network_interface_sr.c

```diff
@@ -5094,7 +5094,7 @@ event_log_temp_expand_pages (THREAD_ENTRY * thread_p, EXECUTION_INFO * info)
 
   tran_index = LOG_FIND_THREAD_TRAN_INDEX (thread_p);
   tdes = LOG_FIND_TDES (tran_index);
-  log_fp = event_log_start (thread_p, "TEMP_VOLUME_EXPAND");
+  log_fp = event_log_start (thread_p, "ADD/EXPAND_VOLUME_INFO");
```

Comment:
EXTEND = EXPAND or ADD
```suggestion
  log_fp = event_log_start (thread_p, "EXTEND_VOLUME_INFO");
```

Reply:
expand가 extend와 add의 상위 개념이라고 생각합니다. EXPAND_VOLUME_INFO로 변경했습니다.

Reply:
다음 콜스택을 기준으로 제안드린 것 입니다.
disk_extend()
  -> disk_volume_expand()
  -> disk_add_volume()